docs/api-inference/getting-started.md

```diff
@@ -0,0 +1,78 @@
+# Getting Started
+
+The Serverless Inference API allows you to easily do inference on a wide range of models and tasks. You can do requests with your favorite tools (Python, cURL, etc). We also provide a Python SDK (`huggingface_hub`) to make it even easier.
+
+We'll do a minimal example using a [sentiment classification model](https://huggingface.co/cardiffnlp/twitter-roberta-base-sentiment-latest). Please visit task-specific parameters and further documentation in our [API Reference](./parameters.md).
+
+## Getting a Token
+
+Using the Serverless Inference API requires passing a user token in the request headers. You can get a token by signing up on the Hugging Face website and then going to the [tokens page](https://huggingface.co/settings/tokens). We recommend creating a `Fine-grained` token with the scope to `Make calls to the serverless Inference API`.
```

Comment:
> We recommend creating a `Fine-grained` token with the scope to `Make calls to the serverless Inference API`.

fyi there's a direct link for this now: https://huggingface.co/settings/tokens/new?globalPermissions=inference.serverless.write&tokenType=fineGrained

Reply:
done in https://github.com/huggingface/hub-docs/pull/1408